### DISCUSSION TITLE
Currently my friend is hosting a site from github pages using a custom domain, I want to change the hosting to my github pages for the same domain

### DISCUSSION THREAD

**Discussion by Rino-RKHTTP** [community/community#60388] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have the goDaddy credentials, Made changes in goDaddy added a C type record to my-username.gihub.io
The page is woking fine from  https://my-username.gihub.io
but I am not able to add custom domain, getting a message Domain is already taken.
Why am i getting this error still after changing the godaddy records?
Should I ask my friend to delete the repo from his end?

**Comment by OshekharO** [comment]
Ask your friend to remove the domain association: If your friend no longer needs the custom domain or is willing to switch to a different domain, they can remove the domain association from their GitHub Pages repository. This will free up the domain for you to use.
